README.md

```diff
@@ -2,7 +2,6 @@
 [![NPM version](https://img.shields.io/npm/v/@brightspace-ui/stylelint-config.svg)](https://www.npmjs.org/package/@brightspace-ui/stylelint-config)
 [![NPM downloads](https://img.shields.io/npm/dt/@brightspace-ui/stylelint-config.svg)](https://www.npmjs.com/package/@brightspace-ui/stylelint-config)
 [![Dependabot badge](https://flat.badgen.net/dependabot/BrightspaceUI/stylelint-config?icon=dependabot)](https://app.dependabot.com/)
-[![Build status](https://travis-ci.com/BrightspaceUI/stylelint-config.svg?branch=master)](https://travis-ci.com/BrightspaceUI/stylelint-config)
```

Comment:
Since there's no CI workflow, it didn't seem to make sense to replace this with the "release" badge.